backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/database/Schema.kt

```diff
@@ -11,9 +11,10 @@ import org.jetbrains.exposed.sql.Op
 import org.jetbrains.exposed.sql.and
 import org.jetbrains.exposed.sql.javatime.datetime
 import org.jetbrains.exposed.sql.or
+import org.jetbrains.exposed.sql.transactions.TransactionManager
 
 object Administrators : IntIdTable() {
-    val email = varchar("email", 100).uniqueIndex()
+    val email = varchar("email", 100)
```

Comment:
Hmm.... In the optimal scenario, we would probably also remove the index that was previously created?

Reply:
Also according to [stackoverflow](https://stackoverflow.com/questions/386294/what-is-the-maximum-length-of-a-valid-email-address) we should up our limit from 100 to at least 254... (a different task for sure)
I feel like we are slowly getting into migrations territory :D ... We should probably think about some proper way to do DB migrations in general in the future...